eos-split-flatpak-repo

```diff
@@ -403,16 +460,31 @@ def main():
 
     logging.basicConfig(level=args.log_level)
 
+    recorder = EosMetrics.EventRecorder.get_default()
+
     try:
-        split_repo(prune=args.prune)
+        with SplitStats() as stats:
+            split_repo(prune=args.prune, stats=stats)
     except Exception as e:
-        recorder = EosMetrics.EventRecorder.get_default()
         recorder.record_event_sync(
             EOS_UPDATER_METRIC_FAILURE,
```

Comment:
I was thinking, hey, we should report a event on error, too. But we do! But… there are no events in this table with component `eos-split-flatpak-repo`. Again I am suspicious of https://phabricator.endlessm.com/T31191…

Reply:
Yes, I see what you mean. This service does start early. Certainly before the event recorder and without any strict ordering w.r.t. to system dbus socket.

Reply:
My debugging in https://phabricator.endlessm.com/T31191 shows that it's an ordering cycle to try to pull in `dbus.socket`. Since in it's current form this service has `eos-updater-flatpak-installer.service`, then it effectively has the same ordering and can't rely on sending the metrics over DBus.

One option is to reorder it after `eos-updater-flatpak-installer.service` and any of the other early boot ostree/flatpak services. The other (I believe) is to write the statistics/errors to disk and have eos-metrics-instrumentation pick them up when it gets started.

Reply:
I opened #362 for the service ordering since I'm haven't gotten to addressing the actual metrics comments.